src/WinUI/ArcGIS.WinUI.Viewer/Samples/Security/IntegratedWindowsAuth/readme.md

```diff
@@ -12,37 +12,33 @@ Your organization might use Integrated Windows Authentication (IWA) to secure Ar
 
 1. Enter the URL to your IWA-secured portal.
 2. Click the button to search for web maps stored on the portal.
-3. You will be prompted for a user name, password, and domain (some platforms will use the current Windows login).
-4. If you authenticate successfully, portal item results will display in the list.
-5. Select a web map item to display it in the map view.
+3. If you authenticate successfully, portal item results will display in the list.
```

Comment:
I'd prefer:

`If authentication is successful portal item results will display in the list.`

Reply:
That's fine but I'm not sure that we need the comma. Your call.